docs-kits/kits/Eco_Pass_KIT/page-software-development-view.md

```diff
@@ -11,43 +13,263 @@ Thereby, this KIT covers various aspects, starting from how the available API En
+ | **/api/contract/agree**               | POST   | Once the user has the contract he can call the `/api/contract/agree` API to start the negotiation process and the transfer of the passport. This means that the user accepted the policy and the frame-contracts contained in the contract policy.                                                                                                                             | [Go to Params](#apicontractagree) |
+ | **/api/contract/decline**             | POST   | The other option rather than `/agree` is the `/decline` API, that basically blocks the process and makes it invalid. This means that the user declined the specific contract that was found for this process.             |                                                                    [Go to Params](#apicontractdecline)                                                                               
+ | **/api/contract/cancel**              | POST   | The user can use `/cancel` to interrupt the negotiation process once it is signed by mistake if is the case. It will be only valid until the negotiation is made.                                                                                                                                                                                                              |    [Go to Params](#apicontractcancel)                                                                                                                           
+ | **/api/contract/status/`<processId>`** | GET    | After the user signs the contract he can use the `/status` API to get the process status and see when it is ready to retrieve the passport using the API `/data`..                                                                                                                                                                                                             |         [Go to Params](#apicontractstatusprocessid)                                                                                                                                                                                                         
+ | **/api/data**                         | POST   | The API `/data` will decrypt the passport file that is encrypted using the session token "sign token", and will delete the file so that it is returned just once to the user and can not be accessed anymore. So a new passport will be always need to be requested..                                                                                                          |  [Go to Params](#apidata)  |
```

Comment:
Just one dot should be enough in "so a new passport will be always need to be requested"

Reply:
ok, weird why it happens